libraries/incubator/ORM/Entity/EntityBuilder.php

```diff
@@ -385,11 +382,12 @@ public function handleStorage(Element $storage)
 					{
 						case 'csv':
 							$handler = '\Joomla\ORM\Storage\CsvProvider';
-							$param   = $parts[1];
+							break;
+						case 'orm':
+							$handler = '\Joomla\ORM\Storage\Doctrine\DoctrineProvider';
```

Comment:
As I can see from the docs (http://doctrine-orm.readthedocs.io/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url), every parameter can be propagated using the DSN URL. Can you give me a use case, where that would not work?